core/orchestrator.py

```
@@ -1,127 +1,278 @@
-            "context": context
+            "context": context,
+            "workflow": workflow_instance,
         }
         return instance_id
-    
-    def _customize_workflow(self, workflow_template: Workflow, custom_parameters: Optional[Dict] = None) -> Workflow:
+
+    def _customize_workflow(
+        self,
+        workflow_template: Workflow,
+        custom_parameters: Optional[Dict] = None,
```

Ruff (0.15.1)

[warning] Unused method argument: `custom_parameters`

(ARG002)